ds3-autogen-cli/src/main/java/com/spectralogic/autogen/cli/CLI.java

```diff
@@ -48,6 +48,16 @@ private Arguments processArgs(final String[] args) throws Exception {
         return arguments;
     }
 
+    private GeneratorType processLanguageArg(final CommandLine cmd) {
+        try {
+            return Guards.returnIfNull(cmd.getOptionValue("l").toUpperCase(), GeneratorType::valueOf);
+        } catch (final NullPointerException e) {
+            return null;
+        } catch (final Exception e) {
```

Comment:
Is there a most specific Exception type that we can use here?  Not all Exceptions will necessarily mean this.